extension-workspace-references.md

```diff
@@ -43,9 +102,9 @@ _Request_
  */
 interface WorkspaceReferencesParams {
     /**
-     * Metadata about the symbol that is being searched for.
+     * Known properties about the identity of the symbol (for example, the package identifier)
      */
-    query: Partial<SymbolDescriptor>;
+    query: Partial<SymbolIdentifier>;
```

Comment:
Yeah, I would like to clarify that. Seems to me like we could define it a bit more clearly as something like
```ts
/** A list of file or directory URIs that the search should be restricted to */
filter: string[]
```